tests/rkt_api_service_test.go

```diff
@@ -153,7 +153,11 @@ func checkPodNetworks(t *testing.T, rawNets map[string]*networkInfo, apiNets []*
 
 // Check the pod's information by 'rkt status'.
 func checkPod(t *testing.T, ctx *testutils.RktRunCtx, p *v1alpha.Pod, hasAppState, hasManifest bool, expectedGCTime time.Time) {
+	t.Logf("API Pod info: %v", p)
```

Comment:
If it helped to debug it this time, maybe next time it flakes it'll help again. I have no problem with more debug output in tests as long as it's not redundant or too spammy (and if we're not running `-v` (I think we're not?) it won't be spammy).